### DISCUSSION TITLE
Split station style from entrance/exit style

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#21016] [Feature requests]
I think it's a good idea to ditch station styles and make them just entrance/exit styles. Ofcourse for backwards compatibility sake station styles should remain a thing, but I think this should become a legacy feature. 

I think ditching them would be smart to make people use scenery instead, which will often look better too, and the station covers will often get in the way.

Even if we wouldn't ditch them i don't think it would harm to split the two.

**Comment by J-PIE-314** [comment]
This is similar to #15742.

**Comment by tygrysek90** [comment]
I have tackled part of this topic by creating [openrct2-open-station](https://github.com/tygrysek90/openrct2-open-stations) parkobj-s - they are ready to use (tho translations are missing, you are welcome to contribute translation, if you want them to use in your native language - details in repository readme)

**openrct2-open-station: The gold classic RCT2 stations with open (not covered) track part**

<img width="426" height="245" alt="Snímek obrazovky z 2025-09-11 18-07-26 ořez malý" src="https://github.com/user-attachments/assets/fe1a753e-9ad8-4db0-aa32-4de7fa8bb522" />

_Download available in separate `parkobj` files or zip /  xzipped tarball with all parkobjs in gh repo linked above_

**Reply by karst** [reply]
This is awesome, good work! Hopefully there will some proper implementation for this at some point so that these aren't necessary. Perhaps even a checkbox could do the trick.

**Reply by tygrysek90** [reply]
Thanks! I have read discussions around this topic before creating this and observed a technical challenges posed in said discussions and as i understood that extending stations in something as different style per station is on edge of do-ability in terms of performance, memory, saving etc.

From my point of view, as these seem to be liked, maybe there could be opened discussion on adding them into the game as they are, in their bare simplicity - same way as No entrance/exit and No platforms were added. The way it breaks nothing legacy, only opens a new possibilities to those who want to have them.
